.github/workflows/test-v3.yml

```
@@ -13,12 +13,18 @@ jobs:
   run-tests:
```

```suggestion
  py=${{ matrix.python-version }}, np=${{ matrix.numpy-version }}, deps=${{ matrix.dependency-set }}:
```
This should yield a more readable job name when the matrix is expanded.

ref: https://futurestud.io/tutorials/github-actions-customize-the-job-name